chwifi

```diff
@@ -109,6 +109,8 @@ check_profile() {
     IFS=' '
     # Check user input against profile array, return true if match is found, otherwise false
     for x in "${profiles[@]}"; do
+        x=$(echo "$x" | sed -e 's/^\s*//g')
+        x=$(echo "$x" | sed -e 's/\s*$//g')
```

Comment:
Could you please add a short comment on what these do, and why - such that a source-reader understand why whitespace is stripped at head and tail?

Reply:
fc4df2a fixes this.